src/com/codersbay/MyStackTest.java

```diff
@@ -0,0 +1,66 @@
+package com.codersbay;
+
+import org.junit.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class MyStackTest {
+
+    @Test
+    public void testPushAndSizing() {
+
+        MyStack myStack = new MyStack();
+
+        myStack.push(1);
+        myStack.push(113);
+        assertEquals(myStack.getSize(), 2);
+
+        myStack.push(2);
+        assertEquals(myStack.getSize(), 3);
+
+    }
+
+    @Test
+    public void testPop() throws StackTooSmallException {
+
+        MyStack myStack = new MyStack();
+
+        myStack.push(10);
+        myStack.push(20);
+        myStack.push(30);
+        myStack.push(40);
+
+        int pop = myStack.pop();
+        assertEquals(pop, 40);
+
+    }
+
+    @Test
+    public void testPopWithSpecificIndex() {
+
+        MyStack myStack = new MyStack();
+
+        myStack.push(10);
+        myStack.push(20);
+        myStack.push(30);
+        myStack.push(40);
+
+        int pop = myStack.myStacks.get(1);
```

Comment:
hier rufst du ja gar nicht pop aus und du verifizierst auch nicht das richtige - was eigentlich ein guter test wäre dass `myStack.pop(2)` ein array zurückgibt in dem [30, 40] drinsteht und danach der myStack nur noch eine größe von 2 hat